### PR TITLE
Add "thin" and "format" columns to disk

### PR DESCRIPTION
**Pull request by borod108**
Required for: https://github.com/ManageIQ/manageiq-providers-ovirt/pull/453
Related to: https://bugzilla.redhat.com/show_bug.cgi?id=1726590

### CODE REVIEW

**Comment by borod108** [comment]
@agrare is this ok?
I know now I removed all other providers it is over engineered, but if someone would like to use this as a bases to do the other ones - it is there.

**Comment by agrare** [comment]
Hey @borod108 , if you just add the `:thin` and `:format` columns I'll take care of the data migration for all of the providers for you.  This should simplify it down to a simple `#change` with the `add_column`s

**Comment by borod108** [comment]
@agrare done.

File: db/migrate/20200219125312_add_thin_and_format_to_disk.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class AddThinAndFormatToDisk < ActiveRecord::Migration[5.1]
+  def change
+    add_column :disk, :thin, :boolean
```
Comment (agrare) [inline db/migrate/20200219125312_add_thin_and_format_to_disk.rb]: Ah this needs to be `:disks` not `:disk`

Comment (borod108) [reply db/migrate/20200219125312_add_thin_and_format_to_disk.rb]: done

**Pull request merged**